### PR TITLE
Remove the legacy Elasticsearch template

### PR DESCRIPTION
**Pull request by webmat**
More complete Elasticsearch 6 and 7 templates are available in generated/elasticsearch.

### CODE REVIEW

**Comment by ruflin** [comment]
You should mention this in the changelog.

**Comment by webmat** [comment]
@ruflin Do you think I should be more verbose on this changelog? The "good" ES templates have been available for a long time in `generated/elasticsearch` and those aren't going away.

I'll definitely have a clear mention of the correct files to use in the release notes, however.

**Comment by webmat** [comment]
Will move this changelog entry to "tooling" after https://github.com/elastic/ecs/pull/633

File: CHANGELOG.next.md [hunk 1]
```diff
@@ -39,6 +39,9 @@ Thanks, you're awesome :-) -->
 
 * Changed the order and column names in the csv. #621
 * Removed the file `schema.json` and the code generating it. #627
+* Removed the legacy Elasticsearch template. #629
+  * Note: The *good* Elasticsearch templates are available in directory
+    `generated/elasticsearch`, this PR only removes an obsolete file.
```
Comment (webmat) [inline CHANGELOG.next.md]: @ruflin What do you think of this clarification?

**Pull request merged**